application/org.openjdk.jmc.flightrecorder.ui/src/main/java/org/openjdk/jmc/flightrecorder/ui/messages/internal/Messages.java

```diff
@@ -630,6 +631,16 @@ public static String stackTraceMessage(int itemCount, int totalCount, String fra
 		return NLS.bind(message, new Object[] {itemCount, frameFraction, totalCount});
 	}
 
+	public static String stackTraceMessage(String itemCount, String totalCount, String frameFraction) {
+		String message;
+		if (itemCount.equals("1")) {
+			message = Messages.STACKTRACE_VIEW_TRACE_OF_TOTAL;
+		} else {
+			message = Messages.STACKTRACE_VIEW_TRACES_OF_TOTAL;
+		}
+		return NLS.bind(message, new Object[] {itemCount, frameFraction, totalCount});
+	}
```

Comment:
**question:** Is this method used ?

Reply:
It was unused method. It was added as part of in progress development. I have cleaned it up now. Thanks for noticing.